crates/viewer/re_view_spatial/src/contexts/transform_tree_context.rs

```diff
@@ -163,7 +163,6 @@ fn lookup_image_plane_distance(
                 )
                 .get_mono_with_fallback::<ImagePlaneDistance>(
```

Comment:
will that one still find the fallback that is registered on Pinhole? Bit confused about that.